packages/story-editor/src/components/library/panes/media/common/mediaElement.js

```diff
@@ -224,7 +227,9 @@ function Element({
           active={active}
         />
         {attribution}
-        {(local || isTranscoding || isMuting || isTrimming) && (
+        {(!src ||
+          isBlobURL(src) ||
```

Comment:
When a empty placeholder is added to state, I wasn't see the progress bar. Instead of local, we need to check for both empty and blob url states.